br/pkg/restore/split.go

```diff
@@ -82,7 +81,7 @@ func (rs *RegionSplitter) ExecuteSplit(
 
 	// Sort the range for getting the min and max key of the ranges
 	// TODO: this sort may not needed if we sort tables after creatation outside.
-	sortedRanges, errSplit := SortRanges(ranges, rewriteRules)
+	sortedRanges, errSplit := SortRanges(ranges)
```

Comment:
I think we have the structure before:
```
map [new table id] ranges
```
Now it is flatten to []ranges.
Actually, we can sort by `new table id` instead of range's key

Reply:
maybe it has no performance issue.